views/js/qtiCommonRenderer/helpers/SelectPointGraphic.js

```diff
@@ -0,0 +1,343 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ */
+
+define([
+    'jquery',
+    'lodash',
+    'raphael',
+    'scale.raphael',
+    'json!taoQtiItem/qtiCommonRenderer/renderers/graphic-style.json'
+], function($, _, raphael, scaleRaphael, gstyle) {
+    'use strict';
+
+
+    /**
+     * Graphic interaction helper
+     * @exports qtiCommonRenderer/helpers/SelectPointGraphicHelper
+     */
+    var SelectPointGraphicHelper = {
+
+        /**
+         * Raw access to the image width
+         * @type {Number}
+         */
+        _width : 0,
```

Comment:
An helper can't keep a state within itself. The same instance of the helper will be shared, for example if you have 2 interactions in the same item, you'll have side effects.